spec/logrotate_spec.rb

```diff
@@ -1,11 +1,7 @@
-require 'spec_helper'
```

Comment:
Yeah, so, this is only needed if you run `rspec` (or `bundle exec rspec`) from the CLI, otherwise, all of our tooling is setup to use the `--require spec_helper` command line option.

I started off just running `rspec` and expecting it to work, and when it bombed out, I added the `require spec_helper` in a couple of places before I realized that it was compensated for elsewhere (e.g. Guardfile, Thorfile, etc).